### ISSUE TITLE
Tab order on race pages should be Fastest, Best Average, Most Runs

### ISSUE THREAD

**Issue by adamengst** [FingerLakesRunnersClub/ChallengeDashboard#18]
Minor point, but everything I've said uses the order of Fastest, Best Average, Most Runs, so it would be good if the tabs on the race page could match that. Thanks!

**Comment by stevedesmond-ca** [comment]
Fixed as of 8a5d354
